File: chihun-jang/예산.js
function solution(d, budget) {
  var answer = 0;
  var want_budget = 0;
  d.sort((a, b) => a - b);
  d.map((item) => {
    want_budget += item;
    if (want_budget <= budget) {
      answer += 1;
    }
  });
  return answer;
}

File: zzzinho/예산.cpp
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> d, int budget) {
    int answer = 0;
    sort(d.begin(), d.end());
    for(int i = 0; i < d.size(); ++i){
        if(budget >= d[i]){
            budget -= d[i];
            ++answer;
        }
    }
    return answer;
}